oshdb-util/src/main/java/org/heigit/bigspatialdata/oshdb/util/changesetenricher/ChangesetEnricher.java

```diff
@@ -0,0 +1,164 @@
+package org.heigit.bigspatialdata.oshdb.util.changesetenricher;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.heigit.bigspatialdata.oshdb.util.OSHDBBoundingBox;
+import org.heigit.bigspatialdata.oshdb.util.OSHDBTimestamp;
+import org.slf4j.LoggerFactory;
+
+public class ChangesetEnricher {
+  private static final org.slf4j.Logger LOG = LoggerFactory.getLogger(ChangesetEnricher.class);
```

Comment:
any reason not to import org.slf4j.Logger?

Reply:
thanks for spotting, was not handling my IDE correctly